chapters/peas-and-pods.rst

```diff
@@ -0,0 +1,123 @@
+Development Guide: Peas and Pods in Jina
+=========================================
+
+.. meta::
+   :description: Development Guide: Peas and Pods in Jina
+   :keywords: Jina, pea, pod
+
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
+.. note:: This guide assumes you have a basic understanding of Parallelization inside Jina, if you haven't, please check out `Parallelization in Jina <../parallel>`_ first.
+
+You might already learned from `Jina 101 <../101.rst>`_.
+Jina :term:`Pea` wraps a :term:`Driver`, :term:`Executor` and lets it exchange data with other Peas.
+Jina :term:`Pod` is a context manager for one or multiple Peas that have the same properties.
+It coordinates Peas to improve efficiency and scaling.
+Beyond that, a Pod adds further control, scheduling, and context management to its Peas.
+
+In this tutorial, we dive deeper into Peas and Pods,
+to help you gain in depth knowledge on the architecture and design principle of Peas and Pods in Jina.
+
+.. contents:: Table of Contents
+    :depth: 3
+
+Pea and Pod Explained
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Singleton Or Not?
+------------------
+
+In jina, if a Pod only contains one Pea, we call it a Singleton Pod.
+If a user, define `parallel` as 1, then it will be considered as Singleton.
+Otherwise, one Pod manages at least two Peas. e.g. `parallel` is at least 2.
```

Comment:
Put reference to the existing docs on parallelism in Jina

Reply:
@cristianmtr referenced at the top of the blog: 
![image](https://user-images.githubusercontent.com/9794489/111574720-4e703300-87ad-11eb-82a5-9033d88fc4ab.png)